templates/directory/organisation_detail.html

```diff
@@ -1,13 +1,97 @@
 {% extends 'base.html' %}
 
+{% load thumbnail %}
+
+{% block extra_head %}
+  {% if organisation.colour %}
+    <style>
+      body {
+        --page-group: {{ organisation.colour }};
+      }
+    </style>
+  {% endif %}
+{% endblock extra_head %}
 
 {% block main_content %}
-  <p>{{ organisation.get_total_private_resources_count }}</p>
-
-  <p>{{ object.get_latest_resource }}</p>
-  <p>{{ object.get_most_liked_resource }}</p>
-  <p>{{ object.get_most_tried_resource }}</p>
-  {% for person in people %}
-    {{ person.get_full_name }}
-  {% endfor %}
+  <div class="profile-summary">
+    <div class="profile-summary__image">
+      {% thumbnail organisation.logo '1000' as thumb %}
+          <div class="profile-image" style="background: url({{ thumb.url }}) center center/contain no-repeat"></div>
+      {% endthumbnail %}
+      <div class="profile-image__details">
+        <h3 class="profile-image__title">{{ organisation.title }}</h3>
+        {{ organisation.description|linebreaks }}
+      </div>
+    </div>
+    <div class="profile-summary__details">
+      <div class="profile-summary__identity">
+        <h1 class="profile-summary__title">{{ organisation.title }}</h1>
+      </div>
+      <div class="profile-details">
+        <div class="icon-list">
```

Comment:
This is very minor but the alignment of the icons and text is slightly off. Line-height or similar fix might be nice, but not necessary